sysconfigs/daint/compilers.yaml

```diff
@@ -182,16 +141,4 @@ compilers:
     modules: []
     environment: {}
     extra_rpaths: []
-- compiler:
```

Comment:
I don't know if we really have to remove them since the frontend is using this os but if you insist